src/commands/subdomain.rs

```diff
@@ -89,12 +94,21 @@ pub fn subdomain(name: &str, user: &GlobalUser, project: &Project) -> Result<(),
         let res_json: Response = serde_json::from_str(&res_text)?;
         if already_has_subdomain(res_json.errors) {
             let sd = Subdomain::get(account_id, user)?;
-            msg = format!(
-                "⛔ This account already has a registered subdomain. You can only register one subdomain per account. Your subdomain is {}.workers.dev \n Status Code: {}\n Msg: {}",
-                sd,
-                res.status(),
-                res_text,
-            );
+            if sd == name {
+                println!(
+                    "{} Your subdomain {}.workers.dev has already been registered!",
```

Comment:
can you say more about what this message is trying to communicate? on first glance i read it and it seemed as though it was supposed to be an error message, but the sparkles seemed confusing. i *think* you mean it as an informative message... let's figure out what exactly we're communicating here, would love to hear more about how you thought about it!

Reply:
So this case is for when someone tries to register the subdomain they've already registered. I don't really think we need to return an error message here (which is the current behavior). Perhaps there's a better way to say this...

Reply:
If that's not clear, this message is returned when avery@cloudflare.com has already registered avery.workers.dev and runs `wrangler subdomain avery`

Reply:
I would read this as "Your subdomain has already been registered _(by someone else)_" – is that the intent here? If not, should it be something like "You've already registered the subdomain xyz.workers.dev"?